### ISSUE TITLE
[docs] 404 when I try to edit github.io page

### ISSUE THREAD

**Issue by apaperclip** [blakeblackshear/frigate#595]
**Describe the bug**
The home assistant section on the github.io page does not mention the switches. I was going to edit the page with updates.

Also to make sure I understand correctly, the switich.camera_clips/detect/snapshots send mqtt messages to the frigate topics to disable that role for that camera. I presume there is a recording as well, but I do not see one since I don't enable that role.

**Version of frigate**
blakeblackshear/frigate:0.8.0-rc4-amd64

latest git log entry from the last custom_component  pull I did.
```
commit d174ecd2664314e8b4c116886b668d6390954d68 (HEAD -> master, tag: v1.0.1, origin/master, origin/HEAD)
Author: Blake Blackshear <blakeb@blakeshome.com>
Date:   Sat Jan 23 00:28:27 2021 +0000

    switch to new snapshot endpoint
```

The target of the edit page link is: 
https://github.com/blakeblackshear/frigate/edit/master/docs/docs/usage/api.md

**Comment by paularmstrong** [comment]
This will be resolved when the `release-0.8.0` branch is merged into `master` – hopefully soon. Will defer to @blakeblackshear whether we should update the URL now or just wait to avoid forgetting to undo the change later.

Relevant configuration:
https://github.com/blakeblackshear/frigate/blob/release-0.8.0/docs/docusaurus.config.js#L63

**Comment by paularmstrong** [comment]
@apaperclip if you'd like to contribute, switch that URL to https://github.com/blakeblackshear/frigate/edit/release-0.8.0/docs/docs/usage/api.md
